grabbit/src/main/groovy/com/twcable/grabbit/resources/RootResource.groovy

```diff
@@ -0,0 +1,38 @@
+package com.twcable.grabbit.resources
+
+import org.apache.sling.api.resource.ResourceResolver
+import org.apache.sling.api.resource.SyntheticResource
+
+import javax.annotation.Nonnull
+
+/**
+ * Copyright 2015 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Representation of a Grabbit's Root Resource provided by {@link GrabbitResourceProvider}.
+ * Queried from {@link com.twcable.grabbit.client.servlets.GrabbitRootServlet}
+ */
+class RootResource extends SyntheticResource  {
```

Comment:
I'd add @CompileStatic here so that we can benefit from static type checking; and any static compilation performance benefits now (however small with the code as-is), or in the future without having to circle-back.